src/Discord/Parts/Channel/Message.php

```diff
@@ -210,13 +212,19 @@ public function getMentionsAttribute()
      *
      * @return Member|User The member that sent the message. Will return a User object if it is a PM.
      */
-    public function getAuthorAttribute()
+    public function getAuthorAttribute($type = 0)
```

Comment:
I just did a quick thing because if you have all users/members loaded, and you're in a guild with 1k+ members,  then you do $message->author, it'll search through 1k+ members to get the author attribute.


I think splitting it would be the best course of action though, have 'author' and 'member', or just have 'author' spit out a user and have people go through `$message->channel->guild->members->offsetGet($message->author->id)` for the `member` attribute